debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSchemaChangeEventEmitter.java

```diff
@@ -105,6 +109,11 @@ public void emitSchemaChangeEvent(Receiver receiver) throws InterruptedException
                         case DROP_TABLE:
                             changeEvents.add(dropTableEvent(partition, tableBefore, (TableDroppedEvent) event));
                             break;
+                        case TRUNCATE_TABLE:
+                            if (truncateReceiver != null) {
+                                truncateReceiver.processTruncateEvent();
+                            }
```

Comment:
```suggestion
                            truncateReceiver.processTruncateEvent();
```

We only ever should receive that event when having the receiver.